config/prod_config.yaml

```diff
@@ -0,0 +1,45 @@
+server:
+  env: "debug"
+  address: "127.0.0.1"
```

Comment:
If this a production config, we need modify some params.
   env: release
   address: 0.0.0.0